verification.py

```diff
@@ -1775,92 +1775,105 @@ def verify_breakpoint_distribution(
                     sample_size=sample_size, initial_size=Ne, growth_rate=growth_rate
                 )
             ],
+            ploidy=ploidy,
             model=model,
         )
+        replicates = sim.run_replicates(1)
```

Comment:
Slightly clearer here if you say 
```
ts = next(sim.run_replicates(1))
```
rather than going through a for loop with one iteration.